modules/user_passwords/utils/common.ts

```diff
@@ -0,0 +1,85 @@
+import base64 from "https://deno.land/x/b64@1.1.28/src/base64.js";
+
+import { createHash as hashSCrypt } from "./scrypt.ts";
+import { createHash as hashBCrypt } from "./bcrypt.ts";
+
+import { hashMatches as matchesSCrypt } from "./scrypt.ts";
+import { hashMatches as matchesBCrypt } from "./bcrypt.ts";
+
+// NOTE: Argon2 is temporarily disabled due to its reliance on `Deno.dlopen`
```

Comment:
make a linear issue for this and link to the gh issue

Reply:
I would do that, but I fixed the problem already.

Reply:
absolute chad energy